samples/callbacks/Version_3/WebSender/App_Start/ServiceBus.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace WebSender
+{
+    using NServiceBus;
+    using NServiceBus.Installation.Environments;
+
+    public class ServiceBus
+    {
+        public static IBus Bus { get; private set; }
+
+        static readonly object padlock = new object();
+
+        public static void Init()
+        {
+            if (Bus != null)
+                return;
+
+            lock (padlock)
```

Comment:
I've always used this pattern in web applications. Requiring an Init() referenced from the Global.asax, to me, follows principle of least surprise - you can look at Global.asax and know all the things that are happening, and get to choose when it happens. Static constructor is decidedly less deterministic.